src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -26,7 +28,7 @@
  *
  * @author Jérémy Derussé <jeremy@derusse.com>
  */
-class CombinedStore implements StoreInterface, LoggerAwareInterface
```

Comment:
This is a BC break for code like:
```
class Service 
{
  public function __construct(StoreInterface $store)
  {
  }
}
```

Could be fixed in FrameworkExtension by decorating the store with a `LegacyStore` which trigger the deprecation